doc/README.adoc

```diff
@@ -268,7 +268,7 @@ CAUTION: This tool is experimental, it may change or be dropped in future.
 .Run a web server to vizualize results.
 [source,terminal,subs=attributes+]
 ----
-korrel8rcli web {korrel8rcli-url} --addr :9090
+korrel8rcli web -u {korrel8rcli-url} --addr :9090
```

Comment:
This seems to work only after client `0.0.4`. @alanconway it seems 0.0.4 wasn't pushed to tags https://github.com/korrel8r/client/tags, so `go install` pick up `0.0.3` which doesn't work with either old or new instructions. Mind pushing the tag to fix that?